guides/RestAPIGuide.md

```diff
@@ -1024,45 +1024,45 @@ Note that the name and description fields are read-only.  If you post an item ob
 
 
 ##### Attachment
-```
+```json
 {
-  "uuid" : <uuid>,
-  * "type" : <string>,
-  * "description" : <string>,
-  "viewer" : <string>,
-  "preview" : <boolean>
+  "uuid" : "<uuid>",
+  * "type" : "<string>",
+  * "description" : "<string>",
+  "viewer" : "<string>",
+  "preview" : "<boolean>"
 }
 ```
 
 NOTE: An attachment object may contain additional fields based on the value of the 'type' field.
 The most common attachment type is 'file'.
 
-**File Attachment** 
-```
+**File Attachment**
+```json
 {
-  "uuid" : <uuid>,
+  "uuid" : "<uuid>",
   * "type" : "file",
-  * "description" : <string>,
-  "viewer" : <string>,
-  * "filename" : <string>,
-  "size" : <int>,
-  "md5" : <string>,
-  "thumbFilename" : <string>,
-  "conversion" : <boolean>,
+  * "description" : "<string>",
+  "viewer" : "<string>",
+  * "filename" : "<string>",
+  "size" : "<int>",
+  "md5" : "<string>",
+  "thumbFilename" : "<string>",
+  "conversion" : "<boolean>",
   "links" : {
     "thumbnail" : ".../thumbs/[uuid]/[version]/[attach uuid]",
     "view" : ".../items/[uuid]/[version]/?attachment.uuid=[attach uuid]"
   }
 }
 ```
 **openEQUELLA Resource Attachment**
-```
+```json
 {
   * "type" : "linked-resource",
-  "uuid" : <uuid>,
-  * "description" : <string>,
-  * "itemUuid" : <uuid>,
-  * "itemVersion" : <int>,
+  "uuid" : "<uuid>",
+  * "description" : "<string>",
+  * "itemUuid" : "<uuid>",
+  * "itemVersion" : "<int>",
```

Comment:
Shouldn't need to quote integers.

Reply:
https://github.com/equella/equella.github.io/pull/25#discussion_r234487858